pkg/cmd/cli/cmd/logs.go

```diff
@@ -163,6 +168,13 @@ func (o *OpenShiftLogsOptions) RunLog() error {
 		}
 		return o.runLogsForBuild(build)
 
+	case "dc", "deploymentconfig", "deploymentconfigs":
```

Comment:
true. so...
```
resourceType := "pods"
...
_, kind, err := mapper.VersionAndKindForResource(resourceType)
if err != nil {
	return err
}
resourceType, _ = meta.KindToResource(kind, false)
```
and then only check "pods", "buildconfigs", "builds", "deploymentconfigs", etc?

does VersionAndKindForResource handle singlar resources, or does resourcebuilder do magic to make that work?